recipes/aeron/all/conanfile.py

```diff
@@ -0,0 +1,161 @@
+        cmake.definitions["BUILD_AERON_ARCHIVE_API"] = True
+        cmake.definitions["AERON_ENABLE_NONSTANDARD_OPTIMIZATIONS"] = True
+
+        cmake.configure(build_folder=self._build_subfolder)
+        return cmake
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+        if self.options.build_tests:
+            self.output.info("Running {} tests".format(self.name))
+            source_path = os.path.join(self._build_subfolder, self._source_subfolder)
+            with tools.chdir(source_path):
+                self.run("ctest")
+
+    def package(self):
+        self.copy(pattern="LICENSE", dst="licenses", src=self._source_subfolder)
+        cmake = self._configure_cmake()
+        cmake.install()
+
+        # Files install straight to './include', but we want './include/aeron'
+        old_include_folder = os.path.join(self.package_folder, "include")
+        new_include_folder = os.path.join(self.package_folder, "include", self.name)
+        tools.rmdir(new_include_folder)
+        tools.mkdir(new_include_folder)
+
+        files = os.listdir(old_include_folder)
+        with tools.chdir(self.package_folder):
+            for f in files:
+                shutil.move(os.path.join(old_include_folder, f), new_include_folder)
+
+        # additional resources
+        archive_resources_dir = os.path.join(self._source_subfolder, "aeron-archive", "src", "main", "resources")
+        self.copy("*", dst="res", src=archive_resources_dir)
+
+        # archive client headers
+        archive_include_dir = os.path.join(self._source_subfolder, "aeron-archive", "src", "main", "cpp", "client")
+        self.copy("*.h", dst="include/aeron-archive", src=archive_include_dir)
+
+        libs_folder = os.path.join(self.package_folder, "lib")
+        if self.options.shared:
+            tools.remove_files_by_mask(libs_folder, "libaeron.so")
+            tools.remove_files_by_mask(libs_folder, "aeron.dll")
+            tools.remove_files_by_mask(libs_folder, "libaeron.dylib")
+            tools.remove_files_by_mask(libs_folder, "*.a")
+            tools.remove_files_by_mask(libs_folder, "*.lib")
+        else:
+            tools.remove_files_by_mask(libs_folder, "libaeron_static.a")
+            tools.remove_files_by_mask(libs_folder, "aeron.lib")
+            tools.remove_files_by_mask(libs_folder, "aeron_static.lib")
+            tools.remove_files_by_mask(libs_folder, "aeron_client_shared.lib")
+            tools.remove_files_by_mask(libs_folder, "*.dll")
+            tools.remove_files_by_mask(libs_folder, "*.so")
+            tools.remove_files_by_mask(libs_folder, "*.dylib")
+
+    def package_info(self):
+        self.cpp_info.libs = tools.collect_libs(self)
+        self.cpp_info.includedirs.append("include/{}".format(self.name))
+
+        self.output.info("{} requires C++11. Enforcing for downstream targets...".format(self.name))
+        self.cpp_info.cppflags.append("-std=c++11")
+
```

Comment:
```suggestion
```

Nope, users should do it. Take a look on other test_packages available in CCI

Reply:
Moved to `validate`.